source/install-guide/configuring-database.rst

```diff
@@ -4,32 +4,105 @@
 Configuring the database
 ########################
 
-This section describes how to configure the database connection, if required.
+Introduction
+------------
+
+GeoNetwork uses a database to persist aspects such as metadata records, privileges and configurations.
+The database default structure is created by the application on initial startup. Subsequent releases of GeoNetwork 
+will update the database structure automatically. For this reason the database user initially needs create privileges on the database. 
+A number of database dialects are supported; :command:`PostGreSQL`, :command:`Oracle`, :command:`SQL Server`, :command:`H2`.
+This section describes various options to configure the database connection.
```

Comment:
Thanks for using command directive :)

Reply:
That should be `PostgreSQL` above.

Is it possible to have a section for each of the databases, what follows has a section for H2 database, but then switches to describing how to configure rather than what to configure.

Consider:
```
PostgreSQL Database
----------------------

How to configure using :file:`jdbc.properties`:
1. step one
2. step two

How to configure using tomcat JNDI:
1. step one
2. step two

How to configure using jetty JNDI:
1. step one
2. step two
```

Reply:
Jody, the setup for postgres, oracle and msql is quite similar, but i agree the H2 default option arrives a bit unexpected on the page